docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -0,0 +1,492 @@
+# ADR 050: SIGN_MODE_TEXTUAL
+
+## Changelog
+
+- Dec 06, 2021: Initial Draft.
+- Feb 07, 2022: Draft read and concept-ACKed by the Ledger team.
+
+## Status
+
+Draft. Concept ACK by the Ledger team, but details need to be polished.
+
+## Abstract
+
+This ADR specifies SIGN_MODE_TEXTUAL, a new string-based sign mode that is targetted at signing with hardware devices.
+
+## Context
+
+Protobuf-based SIGN_MODE_DIRECT was introduced in [ADR-020](./adr-020-protobuf-transaction-encoding.md) and is intended to replace SIGN_MODE_LEGACY_AMINO_JSON in most situations, such as mobile wallets and CLI keyrings. However, the [Ledger](https://www.ledger.com/) hardware wallet is still using SIGN_MODE_LEGACY_AMINO_JSON for displaying the sign bytes to the user. Hardware wallets cannot transition to SIGN_MODE_DIRECT as:
```

Comment:
AFAIKT hardware wallets can us SIGN_MODE_DIRECT. They just have a problem with displaying it to the user. Essentially they could display "Sign Tx123".

Reply:
Yes, that's the first bullet point just below. Blind signing is considered a no-go. Do you think I should clarify it, or it's clear enough?

Reply:
I would clarify that hardware wallets could use _sign mode direct_ with hardware wallets, but it is no-go because ....

Reply:
BTW, I think it's possible to sell a fake hardware wallet which will sign something else then what the users sees (eg different "to_address" for bank.Send).